torch/csrc/jit/passes/xnnpack_rewrite.cpp

```diff
@@ -0,0 +1,121 @@
+#include <torch/csrc/jit/passes/constant_pooling.h>
+#include <torch/csrc/jit/passes/constant_propagation.h>
+#include <torch/csrc/jit/passes/xnnpack_rewrite.h>
+#include <torch/csrc/jit/passes/fuse_linear.h>
+#include <torch/csrc/jit/passes/graph_rewrite_helper.h>
+#include <torch/csrc/jit/passes/subgraph_rewrite.h>
+#include <torch/csrc/jit/ir/subgraph_matcher.h>
+
+namespace torch {
+namespace jit {
+
+namespace {
+std::string getFuncName(Value* func_value) {
```

Comment:
I remember I have this function in quantization.cpp as well

Reply:
Yes, need to move this to the common file too.